node/src/chain_spec/localnet.rs

```diff
@@ -77,6 +77,17 @@ fn localnet_genesis(
             get_account_id_from_seed::<sr25519::Public>("Ferdie"),
             2000000000000u128,
         ),
+        // ETH
+        (
+            // Alith
```

Comment:
please add the ETH address in the doc

Reply:
Not sure where should I add it. Alith and Baltathar are easy to find, but I agree it would be good to have it in our docs too.